src/control/run_go_tests.sh

```diff
@@ -2,6 +2,57 @@
 ## Run linters across control plane code and execute Go tests
 set -eu
 
+function get_cpu_perf()
+{
+	test_file=$(mktemp primebenchXXX.go)
```

Comment:
Any reason to not create the file outright and keep it in utils or something, instead of having the script create and delete it?

Reply:
I started with that approach but it's kind of awkward because it has to be in a `main` package which means its own subdir, which is kinda weird. And I didn't want to put it under `cmd/` because it's not really part of the command surface. Finally, I'm not sure that this information is really all that useful, so I just stashed it here with the thought that we might replace the guts of `get_cpu_perf()` with some other approach or just remove it entirely. This commit is mostly about trying to gather some data so that when people ask me why the unit tests fail occasionally I can look at the results and hopefully start to draw some conclusions based on data rather than hunches.

Reply:
Okay with me. Just struck me as a little weird, thanks for the explanation.